interface/admin.py

```diff
@@ -1,6 +1,49 @@
+from zipfile import ZipFile
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
 from django.contrib import admin
+from django.db.models import F
+from django.http import FileResponse
+
 from interface.models import Course, Assignment, Submission
 
 admin.site.register(Course)
 admin.site.register(Submission)
-admin.site.register(Assignment)
+
+
+@admin.register(Assignment)
+class AssignmentAdmin(admin.ModelAdmin):
+    actions = ['download_submissions']
+
+    def download_submissions(self, request, _assignment):
+        if _assignment.count() != 1:
+            raise RuntimeError('Only one assignment can be selected')
```

Comment:
Ah, but if the app runs with `DEBUG = False`, the user will see a 500 error with no explanation. The polite thing to do is to set a [message](https://docs.djangoproject.com/en/2.2/ref/contrib/messages/) and simply return (I believe Django will redirect the user back to the admin page where they came from).